staff/asix/full-task-app-1.0.0/src/axios.js

```diff
@@ -0,0 +1,27 @@
+const axios = require('axios')
+const baseUrl = 'http://localhost:5000/api/tasks'
+
+const axiosCall = {
+
+    list(res){
+        axios.get(`${baseUrl}`)
```

Comment:
As for responsability purposes I would include render on index.js;

`list(res {
return axios.get(`${baseUrl}`)
}
`